streams/dshr_strdata_mod.F90

```diff
@@ -487,7 +517,8 @@ subroutine shr_strdata_init(sdat, model_clock, stream_name, rc)
              if (chkerr(rc,__LINE__,u_FILE_u)) return
              if (mainproc) then
                 if (i == 1) then
-                   write(sdat%stream(1)%logunit,'(a,i8)') "    adding field "//trim(sdat%pstrm(ns)%fldlist_model(nfld))//&
+                   write(logout,'(2a,i0)') trim(subname),&
+                        "    adding field "//trim(sdat%pstrm(ns)%fldlist_model(nfld))//&
                         " to fldbun_data for stream ",ns
```

Comment:
It would be less awkward if the fields were counted instead of concatenated (throughout file):
```suggestion
write(logout,'(4a,i0)') trim(subname),&
                        "    adding field ", trim(sdat%pstrm(ns)%fldlist_model(nfld)), &
                        " to fldbun_data for stream ",ns
```

Reply:
done